.github/workflows/main-ci.yml

```diff
@@ -3,9 +3,9 @@ name: CI - Main
 on:
   push:
     branches-ignore:
-       - 'skip-ci-*'
-       - 'DEFEDIT-*'
-       - 'editor-dev'
+      - 'skip-ci-*'
+      - 'DEFEDIT-*'
+      - 'editor-dev'
```

Comment:
In order to more easily find the yaml errors locally, before submitting to github, I had to tidy things up (mostly indentations)

I used `yamllint` to do the check locally.
We might want to do that as a githuib hook in the future, to make avoid longer turnaround times.